examples/animatediff/ad/modules/encoders/modules.py

```diff
@@ -47,7 +48,22 @@ def __init__(
             dtype=self.dtype,
         )
 
+    def _clip_tokenize(self, texts):
+        batch_encoding = self.tokenizer(
+            texts,
+            truncation=True,
+            max_length=self.context_length,
+            return_length=True,
+            return_overflowing_tokens=False,
+            padding="max_length",
+        )
+        tokens = ms.Tensor(batch_encoding["input_ids"], ms.int32)
+        return tokens
+
     def tokenize(self, texts):
+        if self.tokenizer_name == "CLIPTokenizer":
```

Comment:
If `self.tokenize` now returns `np.ndarray` instead of `ms.Tensor`, so does `self._clip_tokenize`.

Reply:
fixed